ballerina/constants.bal

```diff
@@ -97,3 +139,6 @@ const string OPPORTUNITY = "Opportunity";
 
 # Constant field `PRODUCT`. Holds the value Product2 for product object.
 const string PRODUCT = "Product2";
+
+# Constant field `NEW_LINE`. Holds the value of "\n".
+const string NEW_LINE = "\n";
```

Comment:
```suggestion
const string NEW_LINE = "\n";

```

Reply:
Updated in [update code according to comments](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/cd6ae536defa51bd247b84742f1c480c2cf80ab1)

Reply:
Updated in [update code according to comments](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/cd6ae536defa51bd247b84742f1c480c2cf80ab1)